benchmarks/benchmarks/stats.py

```diff
@@ -56,7 +56,97 @@ def time_ttest_ind_diff_var(self):
         stats.ttest_ind(self.a, self.c, equal_var=False)
 
 
+class DistributionsAll(Benchmark):
+    # all distributions are in this list. A conversion to a set is used to
+    # remove duplicates that appear twince in either `distcont` or
+    # `distdiscrete`.
+    dists = sorted(list(set([d[0] for d in distcont + distdiscrete])))
+
+    param_names = ['distribution', 'method']
+    params = [
+        dists, ['pdf/pmf', 'logpdf/logpmf', 'cdf', 'logcdf', 'rvs', 'fit',
+                'sf', 'logsf', 'ppf', 'isf', 'moment', 'stats_s', 'stats_v',
+                'stats_m', 'stats_k', 'stats_mvsk', 'entropy']
+    ]
+    dist_data = dict(distcont + distdiscrete)
```

Comment:
I know we've gone through a few iterations on this name. Maybe `dist_shapes` would be the most obvious.